packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -186,6 +212,7 @@ export class AtomicSearchBox {
       getSuggestions: () => this.suggestions,
     };
   }
+  private updateBreakpoints = once(() => updateBreakpoints(this.host));
```

Comment:
Does this do anything? I don't see a mobile or desktop CSS selector.

Reply:
![Screenshot 2022-06-08 at 13 49 34](https://user-images.githubusercontent.com/30511433/172620441-68037a22-95bf-4da2-9a74-852ab7071f09.png)
yeah, it's for this bit in `atomic-search-box` styles